testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -1,3 +1,12 @@
 {
-    "acceptedApiChanges": []
-}
\ No newline at end of file
+    "acceptedApiChanges": [
+        {
+            "type": "org.gradle.api.invocation.Gradle",
+            "member": "Method org.gradle.api.invocation.Gradle.getBuildPath()",
```

Comment:
I chose to introduce this without incubation since the goal is to move existing third party plugins onto this method asap, allowing us to unblock other work